cfgov/unprocessed/css/atoms/partially-styled-link.less

```diff
@@ -0,0 +1,50 @@
+/* topdoc
+  name: Partially styled link
+  family: cfgov-atoms
+  patterns:
+    - name: Default example
+      markup: |
+          <a class="a-partially-styled-link">
+            <span class="a-partially-styled-link_unstyled">
+              Unstyled text
+            </span>
+            <span class="a-partially-styled-link_styled">
+              Underlined text
+            </span>
+          </a>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          a.a-partially-styled-link
+            span.a-partially-styled-link_unstyled
+            span.a-partially-styled-link_styled
+  tags:
+    - cfgov-atoms
+*/
+
+.a-partially-styled-link {
+  border-bottom-width: 0px;
+  
+  .a-partially-styled-link_unstyled {
+    color: @black;
+    border-bottom: 1px dotted transparent;
+  }
+  
+  .a-partially-styled-link_styled {
+    border-bottom: 1px dotted @link-underline;
```

Comment:
The `styled` part of the link (`Read answer` in the first link) should have a dotted underline when the link isn't being interacted with, and then when it's hovered, the whole thing gets a solid underline.